app/javascript/application.js

```diff
@@ -1,3 +1,3 @@
-// Entry point for the build script in your package.json
 import "@hotwired/turbo-rails"
 import "./controllers"
+import "./react/src/index.js"
```

Comment:
- [ ] I see you imported the index, but I don't think it would work if imported but not used. I suggest you use this application file as the index file which you will add all the index file content as we do in an ordinary React project, i.e CRA. At the moment, this is what I can see here after setting it up:
![image](https://user-images.githubusercontent.com/86200356/223185633-5720bd95-846a-4c0e-b816-0e966c64b092.png)

The error message has this to say:
![image](https://user-images.githubusercontent.com/86200356/223186087-aa2bfe17-e77c-48d8-bcd5-dffc47aa782d.png)

- Kudos 👏🏼, your database works
![image](https://user-images.githubusercontent.com/86200356/223188870-4389adee-bbdf-4b4b-a00e-d54db6878487.png)

Reply:
Hi @Olamarx 

I believe you forgot the `yarn build` command

If you want to make changes: `yarn build --watch`